tempesta_fw/t/functional/helpers/chains.py

```diff
@@ -197,6 +217,34 @@ def base(uri='/', method='GET', forward=True, date=None):
                                       server_response=backend_resp)
     return copy.copy(base_chain)
 
+def response_403(date=None, connection=None):
```

Comment:
There is `response_500()` in the begging of the file, and ` response_403` here. Better to group code somehow.

Reply:
fixed